boards/mediatek/index.rst

```diff
@@ -0,0 +1,166 @@
+DRAM area, a convention it inherits from SOF (these devices have
+comparatively large caches which are used for all accesses, unlike
+with intel_adsp).  SRAM is used for interrupt vectors and stacks,
+currently.
+
+There is comparatively little on-device hardware.  The architecture is
+that interaction with the off-chip audio hardware (e.g. I2S codecs,
+DMIC inputs, etc...) is managed by the host kernel.  The DSP receives
+its data via a single array of custom DMA controllers.
+
+Beyond that the Zephyr-visible hardware is limited to a bounty of
+timer devices (of which Zephyr uses two), and a "mailbox"
+bidirectional interrupt source it uses to communicate with the host
+kernel.
+
+Programming and Debugging
+*************************
+
+These devices work entirely in RAM, so there is no "flash" process as
+such.  Their memory state is initialized by the host Linux
+environment.  This process works under the control of a
+``mtk_adsp_load.py`` python script, which has no dependencies outside
+the standard library and can be run (as root, of course) on any
+reasonably compatible Linux environment with a Python 3.8 or later
+interpreter.  A chromebook in development mode with the dev packages
+installed works great.  See the ChromiumOS developer library for more
+detail:
+
+* `Developer mode <https://www.chromium.org/chromium-os/developer-library/guides/device/developer-mode/>`__
+* `Dev-Install: Installing Developer and Test packages onto a Chrome OS device <https://www.chromium.org/chromium-os/developer-library/guides/device/install-software-on-base-images/>`__
+
+Once you have the device set up, the process is as simple as copying
+the ``zephyr.img`` file from the build directory to the device
+(typically via ssh) and running it with the script.  For example for
+my mt8186 device named "steelix":
+
+.. code-block:: console
+
+   user@dev_host:~$ west build -b mt8186//adsp samples/hello_world
+   ...
+   ... # build output
+   ...
+   user@dev_host:~$ scp build/zephyr/zephyr.img root@steelix:
+   user@dev_host:~$ scp soc/mediatek/mt8xxx/mtk_adsp_load.py root@steelix:
+   user@dev_host:~$ ssh steelix
+
+   root@steelix:~ # ./mtk_adsp_load.py load zephyr.img
+   *** Booting Zephyr OS build v3.6.0-5820-gd2a89b3c089e ***
+   Hello World! mt8186_adsp/mt8186_adsp
+
+Debugging
+=========
+
+Given the limited I/O facilities, debugging support remains limited on
+these platforms.  Users with access to hardware-level debug and trace
+tools (e.g. from Cadence) will be able to use them as-is.  Zephyr
+debugging itself is limited to printk/logging techniques at the
+moment.  In theory a bidirectional console like winstream can be used
+with gdb_stub, which has support on Xtensa and via the SDK debuggers,
+but this is still unintegrated.
+
+Toolchains
+**********
+
+The MT8195 toolchain is already part of the Zephyr SDK, so builds for
+the ``mt8195//adsp`` board should work out of the box simply following
```

Comment:
```suggestion
the ``mt8195/adsp`` board should work out of the box simply following
```
nit: As  above, not a blocker, but has a double slash crept in?

Reply:
Nope, see above. :)  That's the real name, it's shorthand for "mt8195/mt8195/adsp"